### PR TITLE
Brain Station 23 adopted Kafka

### PR DESCRIPTION
**Pull request by sadatrafsan**
Brain Station 23 is software development firm based in Dhaka, Bangladesh. Since its inception in 2006, Brain Station 23 has emerged as the hub of tech professionals, minds and enthusiasts. One of our core solution is providing banking solutions. Banking solutions require bulk data transaction and topic/group based data streaming for further processing where we have successfully implemented Kafka to mitigate the transaction bottlenecks and made the system fault tolerant.

### CODE REVIEW

File: powered-by.html [hunk 1]
```diff
@@ -2,6 +2,12 @@
 <script>
     // powered by items
     var poweredByItems = [
+    {
+        "link": "https://brainstation-23.com",
+        "logo": "bs-23.png",
+        "logoBgColor": "#FFFFFF",
+        "description": "Kafka is used at bulk transaction queue and batch data processing in banking solutions where customer satisfaction relies on most. It is also used in batch data validation and filtering process."
```
Comment (bbejeck) [inline powered-by.html]:
```suggestion
        "description": "Kafka is used as a bulk transaction queue and batch data processing in banking solutions where customer satisfaction relies on most. It is also used in batch data validation and filtering process."
```

**Review by bbejeck — COMMENTED** [review]
Hi @sadatrafsan, thanks for the PR - can you add the image `bs-23.png` to the `images/powered-by` directory?

**Comment by sadatrafsan** [comment]
> Hi @sadatrafsan, thanks for the PR - can you add the image `bs-23.png` to the `images/powered-by` directory?

remote: Permission to apache/kafka-site.git denied to sadatrafsan.
fatal: unable to access 'https://github.com/apache/kafka-site.git/': The requested URL returned error: 403

**Comment by sadatrafsan** [comment]
image added to the instructed folder

**Comment by bbejeck** [comment]
> image added to the instructed folder

~Hi @sadatrafsan - did you forget to commit the image? I still don't see `bs-23.png` on your branch~
Nevermind you have what you need on #439 - you can probably close this PR in favor of it.

**Comment by sadatrafsan** [comment]
solved

**Comment by bbejeck** [comment]
Hi, @sadatrafsan I'm thinking you closed this unintentionally - I'm going to reopen the PR and wait a day then merge this.
If you still want to add this to the `Powered By` you don't need to take any further action.

**Review by bbejeck — APPROVED** [review]

**Comment by bbejeck** [comment]
merged #431 into asf-site

**Pull request merged**